src/api.js

```diff
@@ -99,18 +99,18 @@ async function getAccount (accountIndex, axiosConfig = {}) {
 /**
  * GET request to the /transactions-histroy endpoint. Returns a list of forged transaction based on certain filters
  * @param {String} address - Filter by the address that sent or received the transactions. It can be a Hermez Ethereum address or a Hermez BabyJubJub address
- * @param {Number[]} tokenIds - Array of token IDs as registered in the network
+ * @param {Number} tokenId - token ID as registered in the network
```

Comment:
Just a small thing: `- Token ID as registered in the network` (notice the capitalization in `token` word).